api/apps/v1alpha1/nimservice_types.go

```diff
@@ -158,6 +153,13 @@ type MultiNodeMPIConfig struct {
 	MPIStartTimeout int `json:"mpiStartTimeout"`
 }
 
+type Parallelism struct {
+	// +kubebuilder:default:=1
+	// PP specifies the number of pods to create for the multi-node NIMService.
+	// +kubebuilder:validation:Minimum=1
+	PP int `json:"pp,omitempty"`
```

Comment:
```suggestion
	Pipeline *uint32 `json:"pp,omitempty"`
```
can you call this `Pipeline`? PP doesnt sound good